src/components/ContentBlock.vue

```diff
@@ -49,29 +58,50 @@ export default {
       return {
         query: query,
         variables: {
-          id: this.id
+          id: this.id,
         },
         update(data) {
-          console.log('data', data);
           if (data.hasOwnProperty('nodeQuery') &&
             data.nodeQuery.hasOwnProperty('entities') &&
             data.nodeQuery.entities instanceof Array &&
             data.nodeQuery.entities.length === 1) {
             const entity = data.nodeQuery.entities[0];
+
+            this.componentClass = 'content-block-' + this.id.replace("_", "-");
```

Comment:
ID can be too large sometimes, maybe have custom shorter classes for each id in an object would be better.